app/views/layouts/_flashes.html.erb

```diff
@@ -0,0 +1,8 @@
+<% flash.each do |key, value| %>
+  <% if key == "alert" %>
```

Comment:
I do it like that because it's a devise alert. I'll try to override the devise flash messages!